### PR TITLE
Clarify the usage of domain names in egress network policy

### PR DESCRIPTION
**Pull request by pravisankar**

### CODE REVIEW

**Comment by bfallonf** [comment]
Thanks @pravisankar ! Does this tie into an existing BZ or trello card? Or is this just a touch-up that you saw and fixed?

File: admin_guide/managing_networking.adoc [hunk 1]
```diff
@@ -372,14 +372,16 @@ because the `0.0.0.0/0` rule would be checked first, and it would match and deny
...
-different, and the egress network policy may not be enforced as expected. In the
-above example, suppose `www.foo.com` resolved to `10.11.12.13` and has a DNS TTL
-of one minute, but was later changed to `20.21.22.23`. {product-title} will then
-take up to one minute to adapt to these changes.
+domain returned by the local non-authoritative servers. The pod should also
+resolve the domain from the same local nameservers when necessary, otherwise
+the IP addresses for the domain perceived by the egress network policy controller
+and the pod will be different, and the egress network policy may not be enforced
+as expected. Since egress network policy controller and pod are asynchronously
+polling the same local nameserver, there could be race condition where pod may
```
Comment (danwinship) [inline admin_guide/managing_networking.adoc]: "could be A race condition". (Or "could be race conditionS")

**Review by knobunc — APPROVED** [review]
LGTM after you address the comment from Dan

**Comment by pravisankar** [comment]
@bfallonf  related bug: https://bugzilla.redhat.com/show_bug.cgi?id=1575583

**Comment by bfallonf** [comment]
Thanks @pravisankar . This paragraph goes back to 3.6, so I'll cherry-pick this to the 3.6 branch onwards. Please let me know if this is incorrect.

I'll merge.

**Pull request merged**